Tests/Svg.UnitTests/CssQueryTest.cs

```diff
@@ -73,30 +71,31 @@ private void TestSelectorSpecificity(string selector, int specificity)
         ///A test for GetSpecificity
         ///</summary>
         ///<remarks>Lifted from http://www.smashingmagazine.com/2007/07/27/css-specificity-things-you-should-know/, and http://css-tricks.com/specifics-on-css-specificity/ </remarks>
-        [TestMethod()]
-        public void RunSpecificityTests()
+        [Test]
+		[TestCase("*", 0x0)]

```

Comment:
We recently changed all code to use only spaces instead of tabs (there has been a mixture that led to unreadable diffs). Can you please adapt this? And yes, it makes sense to add a PR template where this is stated (similar to the [contributing guide](https://github.com/vvvv/SVG/blob/master/CONTRIBUTING.md)).

Reply:
Whoops, had my whitespace character display disabled on the machine I used today, otherwise I would have noticed. Isn't there an option to store this kind of settings in the project/solution (naturally this will only work in the regular VS versions)?